docs/appendices/release-notes/5.5.2.rst

```diff
@@ -58,3 +58,6 @@ Fixes
 - Fixed an issue with wrong escaping of backslash in
   :ref:`C-style escaped strings <sql_escape_string_literals>`.
   ``SELECT E'\%'`` used to return ``\%`` instead of ``%``.
+
+- Fixed an issue that caused ``ALTER TABLE DROP COLUMN`` to fail to reject
+  dropping :ref:`system columns <sql_administration_system_columns>`.
```

Comment:
This reads like systems columns can be actually dropped. 
But dropped system column still work. Issue is in confusing "OK". 

Maybe we should emphasize the low severity of the bug and phrase like 

 Fixed an issue that caused ``ALTER TABLE DROP COLUMN`` to falsely report success on dropping system cols
 despite on them working afterwards. Now error message is shown.
 
 Feel free to use different phrasing but let's convey that it's only "false report" thing.

Reply:
+1 for the changelog entry rephrasing.